### ISSUE TITLE
Problem with using Evaluator according to Demo notebook

### ISSUE THREAD

**Issue by PavelAvdeyev** [altoslabs/perturbench#9]
I am encountering several blockers while attempting to run the Evaluator API demo notebook.

1. I tried to run evaluator API demo notebook. The Evaluator class attempts to import the following: `from data.accessors.perturbseq_all_bj_v0 import PerturbSeqAllBJV0 ` . However, PerturbSeqAllBJV0 does not appear to exist within the data folder in the current repository. Could you clarify if this file is missing or if the import path needs updating?
 
2.  After following the recommended installation steps, I am facing persistent ImportError issues in Evaluator class. Specifically, the absolute imports starting from data. In the evaluator class do not seem to resolve correctly in the demo environment.

3. Based on the paper and the codebase, it isn't entirely clear if SciPlex3 and Srivatsan20 refer to the same dataset. Could you please confirm if these are identical or if there are specific differences I should be aware of when running the benchmarks?

**Comment by PavelAvdeyev** [comment]
Hi Yan. 

Thank you for the fixes and for answering my questions. I’ve successfully run several notebooks and training scripts.

Since there are no checkpoints available, what is the best way to reproduce the tables from the paper? For example, running 
`train experiment=neurips2025/norman19/linear_best_params_norman19` 
yielded results that differ slightly from Table 3. My run finished with these metrics:

```
model                    LinearAdditive
rmse_average                   0.057800
rmse_rank_average              0.009596
cosine_pca_average             0.807300
cosine_rank_pca_average        0.013130
cosine_logfc                   0.621900
cosine_rank_logfc              0.045450
r2_score_scores               -0.035960
top_k_recall_scores            0.001333
mmd_pca                        1.334000
mmd_rank_pca                   0.013640 
```

Comparing `cosine_logfc`, `cosine_rank_logfc`, and `mmd_pca` with the "LA" or "linear" rows in Table 3, I noticed a discrepancy. Is there a specific GPU/pytorch/amp configuration on top of provided configs I should use to match the paper's results exactly?

**Comment by PavelAvdeyev** [comment]
Hello Yan, 

Thank you for prompt response and providing seeds . 

I tried to train models with op3 configs. I am experiencing some problems with cpa_op3. Decoder model and linear model worked fine. I attached two screenshots. o3_processed.h5ad is bundled with *.csv file similar to other datasets. 

<img width="906" height="111" alt="Image" src="https://github.com/user-attachments/assets/25fc221d-3cef-4737-8cea-a3716b5eca78" />

<img width="650" height="276" alt="Image" src="https://github.com/user-attachments/assets/01b43833-6a15-49e3-8628-af0ae794b8ea" />

Do I run cpa configs incorrectly for op3? Could you please point it out what I need to modify? 

> Yeah setting rank: True will report rank metrics
Thank you for clarifying it. My question is how to compute MMD score without pca embeddings (i.e., mmd_none in notebooks and MMD metric in the paper). To switch on or off pca, I need to provide "average" as a value for aggregation? I am asking since there is "pca_average" for cosine's aggregation value and there is "pca" for mmd aggregation value. Should I provide "average" as aggregation value for "mmd" metric?

**Comment by PavelAvdeyev** [comment]
Yan, 

I trained models with 5 different seeds [245, 301, 302, 303, 304] since you are mentioning 5 seeds averaging in the paper. (245 is coming from default). I evaluated model using final_test config in two ways. One is for all 5 seeds, another one is just for 4 seeds provided by you: [301, 302, 303, 304]. The dataset is norman19.

5 seed result: 
<img width="1142" height="156" alt="Image" src="https://github.com/user-attachments/assets/1ce76deb-2a33-4d63-a357-da3804f81cab" />

4 seed results: 
<img width="1144" height="156" alt="Image" src="https://github.com/user-attachments/assets/3bfdae5c-bc69-4b8c-9c33-cc85f344efaf" />

**The good** is cosine LogFC metric and RMSE metric are similar to one reported in the paper in both cases (but not exactly). Hooooraaaay! 

**The bad** is DEG recall for linear model is 10-100x lower in both cases. Another example is cosine logFC rank metric for CPA model. It is 10 times higher that reported in the paper (table 3). Are such deviation for rank metrics expected?

**Comment by yanwu2014** [comment]
Glad the logfc and RMSE look similar! Let me take a look at the DEG recall

I'll also fix the CPA configs and add the OP3 config for Biolord, will update when done

**Comment by yanwu2014** [comment]
Ok configs and BioLord model should be fixed, let me know if you run into any other bugs? I'll keep looking at the DEG recall

**Comment by PavelAvdeyev** [comment]
Yan, 

You also probably want to update `src/perturbench/configs/data/op3.yaml` file. There ares some problems there. One is splitter parameter, another is transform parameter. There is also problem with `data_iter_factory `and `data`. If you can give me a correct config for data, it would be super helpful.
